mllib/src/main/scala/org/apache/spark/ml/ann/Layer.scala

```diff
@@ -361,9 +361,15 @@ private[ann] trait TopologyModel extends Serializable {
    * Forward propagation
    *
    * @param data input data
+   * @param includeLastLayer include last layer when computing. In MultilayerPerceptronClassifier,
```

Comment:
This text is unclear.  This phrasing is better: "Include the last layer in the output. In MultilayerPerceptronClassifier, the last layer is always softmax; the last layer of outputs is needed for class predictions, but not for rawPrediction."